File: LeetCode/1832_Check_if_the_Sentence_Is_Pangram/solution.js
/**
 * @param {string} sentence
 * @return {boolean}
 */
var checkIfPangram = function(sentence) {
    let obj={};
    if(sentence.length<26)return false
    for(let i=0;i<sentence.length;i++){
        if(obj[sentence[i]]==undefined)obj[sentence[i]]=1
    }
    for(let i=97;i<123;i++){
        let temp = String.fromCharCode(i)
        if(obj[temp]==undefined)return false
    }
    return true;
};

File: LeetCode/1832_Check_if_the_Sentence_Is_Pangram/solution.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        unordered_map<char, int> obj;
        if (sentence.size() < 26)
            return false;
            for (int i = 0; i < sentence.size(); i++) {
                    obj[sentence[i]] = 1;
            }
        for (int i = 0; i <= 25; i++) {
            char temp ='a'+i;
            if (obj[temp] == 0) return false;
        }
        return true;
    }
};